packages/component/src/Localization/da-DK.js

```diff
@@ -41,43 +41,42 @@ function xMinutesAgo(dateStr) {
   });
 }
 
-function botSaidSomething(avatarInitials, text) {
-  return `Bot ${ avatarInitials } sagde, ${ text }`;
+function botSaidSomething(avatarInitials, text, timestamp) {
```

Comment:
Is this related to your latest change (added `timestamp`)?

I am okay to leave it here if it is not suspicious to you.

Reply:
this was an oversight on my part - thanks for the catch